core/model/modx/sources/modmediasource.class.php

```diff
@@ -452,129 +520,164 @@ public function getObjectsInContainer($path)
     public function getObjectContents($path)
     {
         $properties = $this->getPropertyList();
-        $path = $this->fileHandler->postfixSlash($path);
+        $path = $this->postfixSlash($path);
 
         if (!$this->filesystem->has($path)) {
-            $this->addError('file',$this->xpdo->lexicon('file_err_nf'));
+            $this->addError('file', $this->xpdo->lexicon('file_err_nf'));
         }
 
         try {
             /** @var League\Flysystem\File $file */
             $file = $this->filesystem->get($path);
+        } catch (\xPDO\Exception\Container\NotFoundException $e) {
```

Comment:
is this the right exception type here?

Reply:
Of course, not! Thank you for your attentiveness.